AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/holder/EComponentWithViewSupportHolder.java

```diff
@@ -71,10 +71,14 @@
 
 	public EComponentWithViewSupportHolder(AndroidAnnotationsEnvironment environment, TypeElement annotatedElement) throws Exception {
 		super(environment, annotatedElement);
-		viewNotifierHelper = new ViewNotifierHelper(this);
+		viewNotifierHelper = new ViewNotifierHelper(this, environment);
 		keyEventCallbackMethodsDelegate = new KeyEventCallbackMethodsDelegate<>(this);
 	}
 
+	public IJExpression getFindViewByIdExpression(JVar idParam) {
+		return _null();
```

Comment:
I was considering this. Then we'd have the `return _null()` in `EBeanHolder`. I decided to keep it this way, but with no solid reason. If you prefer to have this abstract I can change it.